### DISCUSSION TITLE
plotting pole figure: The figure is too large for the page and will be cut off.

### DISCUSSION THREAD

**Discussion by jkzhmit** [mtex-toolbox/mtex#1535] [Ask Anything] [answered]
Dear community,

Here is a question - when I try to plot the pole figure generated, matlab says "_**Warning: The figure is too large for the page and will be cut off. Resize the figure, adjust the output size by setting the figure's PaperPosition property,
use the 'print' command with either the  '-bestfit' or '-fillpage' options, or use the 'Best fit' or 'Fill page' options on the 'Print Preview' window._**_ "

And the plot is cut off...

![1661277413747](https://user-images.githubusercontent.com/111907912/186230968-8cc9ef4c-b5be-44c1-b353-a0b6d502e266.png)

Does anyone know how to fix this? Thanks very much!

Best regards,
A naive postdoc

**Comment by filippeof** [comment, marked as answer]
Hi,
Did you try reducing the figure size? e.g. using :
```matlab
newMtexFigure('figSize','small')
%... code to plot your pole figure..
```
best,
Filippe

**Reply by jkzhmit** [reply]
Thanks very much! This works.
